packages/core/src/interfaces/axis-scales.ts

```diff
@@ -14,14 +15,15 @@ export interface AxisOptions {
 	 * you would expect to only have 1 axis (dimension) being used as domain
 	 * Domain usually represents labels, ordinal values, time intervals etc.
 	 */
-	useAsDomain?: boolean;
+	useAsGraphDomain?: boolean;
 	/**
 	 * Whether the Axis should be used as the range
 	 * axis of the chart. In the case of Cartesian Scales
 	 * you would expect to only have 1 axis (dimension) being used as range
 	 * Range usually follows a linear scale
 	 */
-	useAsRange?: boolean;
+	useAsGraphRange?: boolean;
+	domain?: AxisDomain[];
```

Comment:
This is a useful addition, let's just add some documentation to show users how to use it.

Documentation should say what it's used for, what the possible values are and what the defaults are, if the value is not provided, along with some examples. It doesn't have to be longer than any other documentation strings.